### PR TITLE
Lab6 Peer Review

### PR DESCRIPTION
**Pull request by jenniv27**
@williamdolsen

### CODE REVIEW

**Comment by jenniv27** [comment]
3e. Explicitly explain that the distribution is antisymmetric because of the difference in 1 sigma distance from mean

**Comment by williamdolsen** [comment]
For Problem 1 I would reword your statistical question, it reads in a bit of a funky way, however what you're saying I believe is statistically correct.

**Comment by williamdolsen** [comment]
For Problem 2a I think the ylabel for your graph should be Probability Density

**Comment by williamdolsen** [comment]
For your explanation in Problem 2b I would mention that in addition to the distribution being biased w/r/t the injected signal value that the probability of getting the injected signal value is next to zero

**Comment by williamdolsen** [comment]
For Problem 3a I would mention Bayes' Theorem and how it is related to the simulation that you are performing

**Comment by williamdolsen** [comment]
For Problem 3b I would plot on the semi-log plot to be absolutely sure that they match as things can be deceiving on a linear scaled plot.  You could also see if the same gaussian that you used in Problem 2 also fits the semi-log plot well

**Comment by williamdolsen** [comment]
General : for plots throughout I think that your y-axis should be Probability Density rather than just Probability

**Comment by williamdolsen** [comment]
For Problem 3c I would once again plot on a semi-log scale before making a conclusion that you have a Gaussian (we know it will be but should always check the semi-log plot)

**Comment by williamdolsen** [comment]
There is an error in your code when you are calculating the uncertainty:
Screen Shot 2021-11-20 at 11 21 48 AM

You are incrementing / decrementing before you add the current probability to the sum.  Doing this increment first rather than as the very last thing is causing you to miss the the probability associated with your mean value of 150 thus giving you the wrong stopping indices / values for the upper and lower bounds.

**Comment by williamdolsen** [comment]
As you stated in your first comment, you'll want to update part e according to your new changes.  Also my previous comment about the bug in your code should also lead to new updates for part e.

**Comment by williamdolsen** [comment]
Great explanations throughout - good job Jennifer!

**Comment by jenniv27** [comment]
Connect current lab with LHC Higgs Boson search

**Comment by jenniv27** [comment]
@AlexSchuy
